### DISCUSSION TITLE
Install geospatial

### DISCUSSION THREAD

**Discussion by decorouz** [giswqs/geog-312#90] [Q&A] [answered]
The command
```python 
uv pip install --find-links https://girder.github.io/large_image_wheels gdal pyproj
```
raises the following exception:
`Exception: Python bindings of GDAL 3.9.2 require at least libgdal 3.9.2, but 3.3.3 was found`

**Comment by giswqs** [comment, marked as answer]
`uv pip install --find-links https://girder.github.io/large_image_wheels gdal==3.3.3 pyproj`
